src/files.js

```diff
@@ -191,6 +191,7 @@ FOAM_FILES([
   { name: 'foam/u2/view/StringArrayRowView', flags: ['web'] },
   { name: 'foam/u2/borders/NullBorder', flags: ['web'] },
   { name: 'foam/u2/borders/CardBorder', flags: ['web'] },
+  { name: 'foam/u2/borders/SplitScreenBorder', flags: ['web'] },
```

Comment:
This was SplitBorder in business code side. Moved it to foam. Business code branch (https://github.com/nanoPayinc/NANOPAY/pull/8340) deletes legacy file